cmd/security-secrets-setup/main.go

```diff
@@ -16,80 +17,111 @@ package main
 
 import (
 	"flag"
-	"log"
+	"fmt"
 	"os"
+	"path/filepath"
 	"time"
 
 	"github.com/edgexfoundry/edgex-go/internal"
-	"github.com/edgexfoundry/edgex-go/internal/pkg/config"
 	"github.com/edgexfoundry/edgex-go/internal/pkg/usage"
 	"github.com/edgexfoundry/edgex-go/internal/security/setup"
-	"github.com/edgexfoundry/edgex-go/internal/security/setup/certificates"
+	"github.com/edgexfoundry/edgex-go/internal/security/setup/option"
```

Comment:
we should change this folder name to 
"github.com/edgexfoundry/edgex-go/internal/security/**secretssetup**/option" OR
"github.com/edgexfoundry/edgex-go/internal/security/**secrets**/option"
the current name /security/**setup** implies that it includes stuff for proxy-setup and other secretstore-setup. I think that is not the case, correct?

Reply:
the proxy stuff is under different directory of `/security/proxy`. this may need to bring to the group for discussion since it is kind of directory structure changes

Reply:
right, security/proxy and security/secrets are more consistent than what we currently have

Reply:
Agree with Jim.  Can refactor this later.  Don't need additional controversey at this point.